packages/rocketchat-i18n/i18n/en.i18n.json

```diff
@@ -3804,7 +3804,7 @@
   "Teams_delete_team_warning": "This can’t be undone. Once you delete a Team, all chat content and configuration will be deleted.\n\n In the next step, you can choose if you will keep or delete Channels.",
   "Teams_delete_team": "You are about to delete this Team.",
   "Teams_deleted_channels": "The following Channels are going to be deleted:",
-  "Teams_Errors_Already_exists": "The team `__name__` already exists.",
```

Comment:
I think you shouldn't edit the existing translation, but create a new one. If you edit the old one, all other languages will be wrong. Also, the key doesn't match the text.

I know there aren't any other laguages translated right now, but this is expected. Please, change the key to match the text.